src/tsjson-parser.ts

```diff
@@ -19,9 +19,10 @@ export type Validated<T extends SchemaLike> = T[typeof InternalTypeSymbol];
 export class TsjsonParser<T extends SchemaLike> {
   public readonly schema: T;
   private readonly validator: Ajv.ValidateFunction;
-  constructor(schema: T) {
+
+  constructor(schema: T, ajvOptions: Ajv.Options = {}) {
```

Comment:
This does leak out one of Ajv's types, but I don't think `ajv` needs to go in `peerDependencies` since the leaked type is just an `object` of literals.